packages/reactstrap-validation-select/src/AvSelect.js

```diff
@@ -381,6 +397,8 @@ AvSelect.propTypes = {
   loadOptions: PropTypes.func,
   raw: PropTypes.bool,
   creatable: PropTypes.bool,
+  ariaFeedback: PropTypes.object,
```

Comment:
@jordan-a-young do we normally define the object shape if we know it?
ariaFeedback: PropTypes.shape({
feedbackId: PropTypes.string,
errorMessage: PropTypes.string
})

Reply:
Same answer as my other comment. I think we should define it if we know the shape